File: data/src/main/java/com/ykb/app/cryptotrader/data/repo/RoleRepo.java
package com.ykb.app.cryptotrader.data.repo;

import com.ykb.app.cryptotrader.data.model.Role;
import com.ykb.app.cryptotrader.data.model.User;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface RoleRepo extends JpaRepository<Role, String> {
}

File: library/data/data-auth/src/main/kotlin/com/ykb/app/cryptotrader/data/auth/repo/RoleRepo.kt
package com.ykb.app.cryptotrader.data.auth.repo

import com.ykb.app.cryptotrader.data.auth.model.Role
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface RoleRepo : JpaRepository<Role, String>